File: project/app/src/main/java/org/owntracks/android/support/widgets/PausableViewPager.java
package org.owntracks.android.support.widgets;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

import androidx.viewpager.widget.ViewPager;


public class PausableViewPager  extends ViewPager {

    public PausableViewPager(Context context, AttributeSet attrs) {
            super(context, attrs);
        }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent event) {
        return false;
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        return false;
    }

    public void forward() {
        setCurrentItem(getCurrentItem()+1);
    }

}

File: project/app/src/main/java/org/owntracks/android/support/widgets/PausableViewPager.kt
package org.owntracks.android.support.widgets

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import androidx.viewpager.widget.ViewPager

class PausableViewPager(context: Context?, attrs: AttributeSet?) : ViewPager(context!!, attrs) {
    override fun onInterceptTouchEvent(event: MotionEvent): Boolean {
        return false
    }

    override fun onTouchEvent(event: MotionEvent): Boolean {
        performClick()
        return false
    }

    override fun performClick(): Boolean {
        super.performClick()
        return false
    }
}